app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] [REQUIRED] Good job so far! The application asks for `confirmation` of `email, `:confirmable` should be enabled in the `user` model 🙏🏼 
```ruby
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```